trustmanager/remoteks/client.go

```diff
@@ -0,0 +1,94 @@
+package remoteks
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/notary/trustmanager"
+	google_protobuf "github.com/golang/protobuf/ptypes/empty"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// RemoteStore is a wrapper around the GRPC storage client, translating between
+// the Go and GRPC APIs.
+type RemoteStore struct {
+	client   StoreClient
+	location string
+}
+
+var _ trustmanager.Storage = &RemoteStore{}
```

Comment:
Hacky go compile time check to ensure a type implements the desired interface. You'll see it around distribution a lot too. It's relatively idiomatic if unofficial.